wls-gui-wahllokalsystem/src/stores/infomanagementStore.ts

```diff
@@ -49,27 +75,38 @@ export const useInfomanagementStore = defineStore(storeID, () => {
   }
 
   /** FRUEHESTE_EROEFFNUNGSZEIT bezeichnet den frühesten Wert, zu dem die Wahlhandlung eröffnet werden kann. */
-  const fruehesteEroeffnungsuhrzeitUWB = computed(() => {
-    const param = konfigurationsparameter.value?.find(
-      (param) => param.schluessel === "FRUEHESTE_EROEFFNUNGSZEIT_UW"
-    );
-    return param?.wert || "00:00:00";
+  const _fruehesteEroeffnungsuhrzeitUWB = computed(() => {
+    return _getOrDefault("FRUEHESTE_EROEFFNUNGSZEIT_UW", DEFAULT_TIME);
   });
 
   /** FRUEHESTE_SCHLIESSUNGSZEIT bezeichnet den spätesten Wert, zu dem die Wahlhandlung eröffnet werden kann. */
-  const fruehesteSchliessungsuhrzeitUWB = computed(() => {
+  const _fruehesteSchliessungsuhrzeitUWB = computed(() => {
+    return _getOrDefault("FRUEHESTE_SCHLIESSUNGSZEIT_UW", DEFAULT_TIME);
+  });
+
+  /** FRUEHESTE_EROEFFNUNGSZEIT bezeichnet den frühesten Wert, zu dem die Wahlhandlung eröffnet werden kann. */
+  const _fruehesteEroeffnungsuhrzeitBWB = computed(() => {
+    return _getOrDefault("FRUEHESTE_EROEFFNUNGSZEIT_BW", DEFAULT_TIME);
+  });
+
+  /** FRUEHESTE_SCHLIESSUNGSZEIT bezeichnet den spätesten Wert, zu dem die Wahlhandlung eröffnet werden kann. */
+  const _fruehesteSchliessungsuhrzeitBWB = computed(() => {
+    return _getOrDefault("FRUEHESTE_SCHLIESSUNGSZEIT_BW", DEFAULT_TIME);
+  });
+
+  function _getOrDefault(schluessel: string, defaultValue: string) {
```

Comment:
```suggestion
  function _getKonfigParamOrDefault(schluessel: string, defaultValue: string) {
```
finde den namen so zu generisch

Reply:
geht auch `_getKonfigParamWertOrDefault`, da `KonfigParam` das gesamte Objekt ist. Anstelle von `Wert` was die Props ist könnte ich mir auch `value`, passend zum 2. Argument vorstellen.

Reply:
ja geht auch. `value` find ich besser

Reply:
name geändert in `_getKonfigParamValueOrDefault`  mit b16c332e28b8608180f3c8a8de2d411ef7c0231e